tox.ini

```diff
@@ -6,7 +6,7 @@ ignore = E129,W504
 changedir = tests
 deps = pytest
        requests
-commands = pytest
+commands = pytest -v {posargs} -p aiohttp.pytest_plugin -p colibris.test
```

Comment:
Why `-p colibris.test`? That's a package intended for the Colibris-based project tests, not for Colibris itself.

Reply:
Not sure about it, don't we need it? But I'm sure we need `aiohttp.pytest_plugin`.

Reply:
Removed!